handbook/marketing/adding_screenshots_screen_recording.md

```diff
@@ -105,18 +107,84 @@ But to get you started, here are some essential tips:
+After you're done, click "Download" and select "WebVTT (.vtt)" as the format to download.
+HappyScribe also offers automatic and human-transcribed subtitles as a paid service.
+
+Another option is to use an automatic transcription service like [Otter.ai](https://otter.ai/).
+Otter offers a download in `.SRT` format, which can be easily [converted to WebVTT](https://www.happyscribe.com/subtitle-tools/convert-srt-to-vtt).
+You may need to edit the captions afterwards to correct any mistakes, which you can do in [HappyScribe](https://www.happyscribe.com/subtitle-tools/online-subtitle-editor/free) (click "New subtitles" at the top → "OK" → "Select file") or by editing the WebVTT file in a text editor (it's just a text file).
+
+Zoom Cloud recordings give you an automatically-transcribed WebVTT file when you click "Download" in the top right as one of the files being downloaded, which you can use directly as well.
+
+Once you have your `.vtt` file, upload it to [sourcegraph-assets](https://console.cloud.google.com/storage/browser/sourcegraph-assets/?project=sourcegraph-de&folder=true&organizationId=true) next to the video with the same file name (but `.vtt` ending), then embed the video with a **captions track** like so:
+
+```html
+<video controls crossorigin>
+  <source src="https://cors-anywhere.sgdev.org/https://sourcegraphstatic.com/my_video.mp4" />
+  <track
+    default
+    kind="captions"
+    label="Captions"
+    src="https://cors-anywhere.sgdev.org/https://sourcegraphstatic.com/my_video.vtt"
+  />
+</video>
+```
+
+Note that the `crossorigin` attribute needs to be added and the URLs are prefixed with `https://cors-anywhere.sgdev.org/` to work correctly in all browsers.
+
+You can choose to keep or remove the `default` attribute on the `<track>` to control whether the captions should be shown by default or not.
+It is recommended to show them by default, as many people even with good hearing like captions.
+
 ### Recording your voiceover
 
 - If using ScreenFlow 9, you can record audio using the narration feature which can be accessed from the menu bar: **Insert** > **Narration**
 - Use your phone to record your voice-over, then import the audio file into ScreenFlow (produces a better sounding result than using your laptop microphone)
 
-## Embedding GIFs
+## Embedding videos
+
+The video should be uploaded to the [sourcegraph-assets](https://console.cloud.google.com/storage/browser/sourcegraph-assets/?project=sourcegraph-de&folder=true&organizationId=true) on Google Cloud storage and linked to from the URL `https://sourcegraphstatic.com/path/to/file`.
+Use or create subfolders as appropiate, e.g. the `handbook` or `blog` folder.
+
+Videos are embedded in Markdown and HTML using the `<video>` tag:
+
+```html
+<video controls src="https://sourcegraphstatic.com/my-funny-video.mp4"></video>
```

Comment:
Nit: maybe include the entire tag as a template, with the captions track and all?

Reply:
Hmm, I think there are legit cases where a video may not have a caption track, like when the video simply has no audio voice-over. Since it may not be obvious to everyone that you need to remove the `<track>` in that case as not everyone knows HTML, I thought it would be good to have an example for each case (simple videos, GIF-like videos, videos with captions)